reactioncommerce/reaction-next-starterkit
# Feat : Test SSR Render

Resolves #184 
Impact: **minor**
Type: **feature**

## Issue
SSR was broken and it was not realized until a while passed.

## Solution
An automated test that verifies DOM structure to ensure that NextJS is rendered in the HTML hooked in the CI.

```
<div id="__next">HTML_CONTENT_GOES_HERE</div>
```


## Testing
1. Load up the application locally
2. Run `yarn run test:ssr`
3. Verify if test is working properly

Expected Output
```
$ mocha tests/ssr


  NextJS Loading
    √ SSR Loads with an HTML Body


  1 passing (9ms)

Done in 0.89s.
```
